tests/tests.rs

```diff
@@ -1026,25 +1026,31 @@ fn should_decode_address() {
 
 #[test]
 fn should_decode_transaction_receipt() {
-    let value = candid_types::TransactionReceipt {
-        status: 0x1.into(),
+    let value = evm_rpc_types::TransactionReceipt {
+        status: 0x1_u8.into(),
         transaction_hash: "0xdd5d4b18923d7aae953c7996d791118102e889bea37b48a651157a4890e4746f"
-            .to_string(),
+            .parse()
+            .unwrap(),
         contract_address: None,
         block_number: 18_515_371_u64.into(),
         block_hash: "0x5115c07eb1f20a9d6410db0916ed3df626cfdab161d3904f45c8c8b65c90d0be"
-            .to_string(),
+            .parse()
+            .unwrap(),
         effective_gas_price: 26_776_497_782_u64.into(),
-        gas_used: 32_137.into(),
-        from: "0x0aa8ebb6ad5a8e499e550ae2c461197624c6e667".to_string(),
+        gas_used: 32_137_u32.into(),
+        from: "0x0aa8ebb6ad5a8e499e550ae2c461197624c6e667"
+            .parse()
+            .unwrap(),
         logs: vec![],
-        logs_bloom: "0x0".to_string(),
```

Comment:
Note that the previous value
```rust
logs_bloom: "0x0".to_string(),
```
is not valid. `logsBloom` [MUST](https://github.com/ethereum/execution-apis/blob/b7432c54d2faa5f282db7b3202c3c2ef907103ad/src/schemas/receipt.yaml#L105) be `^0x[0-9a-f]{512}$`.

Reply:
I originally shortened `logsBloom` for readability since there wasn't any validation for the returned string, but it definitely makes sense to change this now.